### ISSUE TITLE
TST: ADASS 99 P2-42 gives 404

### ISSUE THREAD

**Issue by pllim** [astropy/astropy#16824]
Linkcheck reported broken https://www.adass.org/adass/proceedings/adass99/P2-42/ (Error 404)

I have contacted the website maintainer and waiting to hear back.

**Comment by teuben** [comment]
indeed,  the adass.org server changed location yesterday. Good catch!

**Comment by pllim** [comment]
Thanks! I might prefer the `aspbooks` URL because it lets me read the paper. I cannot figure out how to access the actual paper from `adsabs` site (also it has less than ideal dark mode support but that is another story).

**Comment by pllim** [comment]
That said, if you plan to restore the original URL anyway, then there is nothing to do here. We will just wait it out. I think it is nice to see from URL that paper came from ADASS.
